core/commands/add.go

```diff
@@ -116,6 +117,7 @@ You can now check what blocks have been created by:
 		cmdkit.BoolOption(trickleOptionName, "t", "Use trickle-dag format for dag generation."),
 		cmdkit.BoolOption(onlyHashOptionName, "n", "Only chunk and hash - do not write to disk."),
 		cmdkit.BoolOption(wrapOptionName, "w", "Wrap files with a directory object."),
+		cmdkit.StringOption(pathName, "Assign path name when use wrap-with-directory option"),
```

Comment:
Let's change this to reflect that the option (with the new check) works only for `stdin`, e.g., "Assign a name if the file source is stdin" (or something like that, feel free to reword it).

Reply:
@schomatis @keks In godoc,the file has a function named `fullpath` not `fullname`.So i use `fullpath` to judge the name, And In my test,if i use command such as
```
$cat ./test1.txt | ../ipfs add -w --name "myfile.txt"
added Qmf3xdREx34pewgYZjyvHQsrZxuk1TLUFdzoF95dsXfv3k myfile.txt
added QmSLquBbWRXs3VjTCJBXYv4BHn7NFNZxh5iU3zGNp5aMcf
```
the file path is also is empty string:
```
09:20:50.138 INFO   coreunix: The file path is : add.go:475
```
not os.Stdin(/dev/stdin).
So please help me review my pr.